doc/collections/_adrs/0004--decision_for_tools_dir.jekyll

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
i am still thinking about this, but check https://github.com/OpenINF/openinf.github.io/pull/1090 for some real-world use cases and half-baked justification